### ISSUE TITLE
Force dot for float values 

### ISSUE THREAD

**Issue by chalda-pnuzig** [PHPOffice/PhpSpreadsheet#226]
This is:

- [x] a bug report

### What is the current behavior?
If the current locale is different from english the excel file is corrupted, because the float values could be "xx,xx" instead of "xx.xx":

### What are the steps to reproduce?

```php
<?php



// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();

/**/

setlocale(LC_ALL, 'en');
$objWriter = new Xlsx($spreadsheet);
$objWriter->setIncludeCharts(TRUE);
$objWriter->save($filename); 
 // CORRECT : <c:v>100.12</c:v>

setlocale(LC_ALL, 'it'); 
$objWriter = new Xlsx($spreadsheet);
$objWriter->setIncludeCharts(TRUE);
$objWriter->save($filename);
// WRONG: <c:v>100,12</c:v>

```

### Which versions of PhpSpreadsheet and PHP are affected?

PHP v. 7.0.1

**Comment by PowerKiKi** [comment]
Code is incomplete and does not allow to reproduce bug.

**Comment by chalda-pnuzig** [comment]
This is a working example to reproduce the issue based on the [sample 33 (33_Chart_create_line.php)](https://github.com/PHPOffice/PhpSpreadsheet/blob/develop/samples/33_Chart_create_line.php)

The problem is verified with Office 2010 Service Pack 2 and newer.

Here are the generated files with the code below: 
- [locale_en.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/1320576/locale_en.xlsx) : no problem
- [locale_it.xlsx](https://github.com/PHPOffice/PhpSpreadsheet/files/1320577/locale_it.xlsx) :  an alert appears with message "the file is corrupted" when opening

```php
<?php
use PhpOffice\PhpSpreadsheet\Chart;
use PhpOffice\PhpSpreadsheet\Chart\DataSeries;
use PhpOffice\PhpSpreadsheet\Chart\DataSeriesValues;
use PhpOffice\PhpSpreadsheet\Chart\Legend;
use PhpOffice\PhpSpreadsheet\Chart\PlotArea;
use PhpOffice\PhpSpreadsheet\Chart\Title;
use PhpOffice\PhpSpreadsheet\Spreadsheet;
use PhpOffice\PhpSpreadsheet\Writer\Xlsx;

require __DIR__ . '/Header.php';

$spreadsheet = new Spreadsheet();
$worksheet = $spreadsheet->getActiveSheet();
$worksheet->fromArray(
    [
            ['', 2010, 2011, 2012],
            ['Q1', 12.100, 15, 21], // decimal number
            ['Q2', 56, 73, 86],
            ['Q3', 52, 61, 69],
            ['Q4', 30, 32, 0],
        ]
);
//	Set the Labels for each data series we want to plot
//		Datatype
//		Cell reference for data
//		Format Code
//		Number of datapoints in series
//		Data values
//		Data Marker
$dataSeriesLabels = [
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_STRING, 'Worksheet!$B$1', null, 1), //	2010
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_STRING, 'Worksheet!$C$1', null, 1), //	2011
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_STRING, 'Worksheet!$D$1', null, 1), //	2012
];
//	Set the X-Axis Labels
//		Datatype
//		Cell reference for data
//		Format Code
//		Number of datapoints in series
//		Data values
//		Data Marker
$xAxisTickValues = [
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_STRING, 'Worksheet!$A$2:$A$5', null, 4), //	Q1 to Q4
];
//	Set the Data values for each data series we want to plot
//		Datatype
//		Cell reference for data
//		Format Code
//		Number of datapoints in series
//		Data values
//		Data Marker
$dataSeriesValues = [
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_NUMBER, 'Worksheet!$B$2:$B$5', null, 4),
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_NUMBER, 'Worksheet!$C$2:$C$5', null, 4),
    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_NUMBER, 'Worksheet!$D$2:$D$5', null, 4),
];
//	Build the dataseries
$series = new DataSeries(
    DataSeries::TYPE_LINECHART, // plotType
    DataSeries::GROUPING_STACKED, // plotGrouping
    range(0, count($dataSeriesValues) - 1), // plotOrder
    $dataSeriesLabels, // plotLabel
    $xAxisTickValues, // plotCategory
    $dataSeriesValues        // plotValues
);
//	Set the series in the plot area
$plotArea = new PlotArea(null, [$series]);
//	Set the chart legend
$legend = new Legend(Legend::POSITION_TOPRIGHT, null, false);
$title = new Title('Test Stacked Line Chart');
$yAxisLabel = new Title('Value ($k)');
//	Create the chart
$chart = new Chart(
    'chart1', // name
    $title, // title
    $legend, // legend
    $plotArea, // plotArea
    true, // plotVisibleOnly
    0, // displayBlanksAs
    null, // xAxisLabel
    $yAxisLabel  // yAxisLabel
);
//	Set the position where the chart should appear in the worksheet
$chart->setTopLeftPosition('A7');
$chart->setBottomRightPosition('H20');
//	Add the chart to the worksheet
$worksheet->addChart($chart);


// Save Excel 2007 file with locale en 
setlocale(LC_ALL, 'en_US.UTF-8', 'en_US', 'en','english');
$filename = $helper->getFilename('locale_en.xlsx');
$writer =  new Xlsx($spreadsheet);
$writer->setIncludeCharts(true);
$callStartTime = microtime(true);
$writer->save($filename);
$helper->logWrite($writer, $filename, $callStartTime);

// Locale with comma as decimal separator
setlocale(LC_ALL, 'it_IT@euro', 'it_IT', 'it', 'italian');

$filename = $helper->getFilename('locale_it.xlsx');
$writer =  new Xlsx($spreadsheet);
$writer->setIncludeCharts(true);
$callStartTime = microtime(true);
$writer->save($filename);
$helper->logWrite($writer, $filename, $callStartTime);
```

**Comment by PowerKiKi** [comment]
I had a closer look. At first I couldn't reproduce because the Italian locale is not available on my machine. Thus `setlocale()` failed silently. Then I actually found out that this is a duplicate of https://github.com/PHPOffice/PHPExcel/issues/1107.

Digging deeper I found that it's the [XMLWriter](http://php.net/manual/en/book.xmlwriter.php) that is writing those specific attributes, for instance [on that line](https://github.com/PHPOffice/PhpSpreadsheet/blob/e6c95bf9b0400b2af07372dde2914c01b7880542/src/PhpSpreadsheet/Writer/Xlsx/Chart.php#L1269). Since it is out of our control, there is unfortunately no better solution than to always be sure to set a neutral locale before using PhpSpreadsheet like so:

```php
setlocale(LC_ALL, 'en_US.UTF-8');
```
